src/nft/utils/listNfts.ts

```diff
@@ -147,21 +145,20 @@ export async function signListing(
                 itemType: ItemType.ERC721,
                 token: asset.asset_contract.address,
                 identifier: asset.tokenId,
-                amount: '1',
               },
             ],
             consideration: considerationItems,
             endTime: asset.expirationTime.toString(),
-            zone: OPENSEA_DEFAULT_ZONE,
-            restrictedByZone: true,
+            zone: ZERO_ADDRESS,
```

Comment:
The tweet recommends doing this however OS's web app still uses the default zone. Currently asking their team for guidance.